server/utils/utils.test.ts

```diff
@@ -21,8 +21,8 @@ describe('convert to title case', () => {
     ['Upper case', 'ROBERT', 'Robert'],
     ['Mixed case', 'RoBErT', 'Robert'],
     ['Multiple words', 'RobeRT SMiTH', 'Robert Smith'],
-    ['Leading spaces', '  RobeRT', '  Robert'],
-    ['Trailing spaces', 'RobeRT  ', 'Robert  '],
```

Comment:
Wasn't sure why we would want these to remain.... please let me know if you know the reason